docs/running-on-kubernetes.md

```diff
@@ -768,6 +768,22 @@ from the other deployment modes. See the [configuration page](configuration.html
     <code>myIdentifier</code>. Multiple node selector keys can be added by setting multiple configurations with this prefix.
   </td>
 </tr>
+<tr>
+  <td><code>spark.executorEnv.[EnvironmentVariableName]</code></td> 
+  <td>(none)</td>
+  <td>
+    Add the environment variable specified by <code>EnvironmentVariableName</code> to
+    the Executor process. The user can specify multiple of these to set multiple environment variables.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.driverEnv.[EnvironmentVariableName]</code></td> 
```

Comment:
On the Yarn `cluster` mode, there's a similar property named `spark.yarn.appMasterEnv.[EnvironmentVariableName]`. Since this is specific to Kubernetes, should we rename it to `spark.kubernetes.driverEnv[EnvironmentVariableName]`?